client/src/components/EditProfile/PaymentTab/CreditCard/CreditCard.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Typography, Checkbox } from '@material-ui/core';
+import useStyles from './useStyles';
+import MastercardLogo from './mastercard.png';
+import VisaLogo from './visa.png';
+
+interface props {
+  card: {
+    name: string;
+    cardNumber: string;
+    cardExpDate: string;
+    cardCVC: string;
+    cardType: string;
+  };
+}
+
+export default function CreditCard(props: props): JSX.Element {
+  const classes = useStyles();
+  const card = props.card;
+
+  return (
+    <Box className={classes.creditCard}>
+      <Box display="flex" justifyContent="space-between">
+        {card.cardType === 'mastercard' && <img className={classes.cardLogo} src={MastercardLogo} />}
+        {card.cardType === 'visa' && <img className={classes.cardLogo} src={VisaLogo} />}
```

Comment:
It's a good practice to add the `alt` attribute to `img` tag as this can be useful to screen readers.